### PR TITLE
I9

### PR DESCRIPTION
**Pull request by Kwuerch**
Merging I9 into Master

### CODE REVIEW

File: olympus/main.c [hunk 1]
```diff
@@ -41,18 +41,31 @@ Timer_Return updateSteerDataLink(void* dummy)
     if(updateHeraSteer() != COMMS_OK)
     {
         setSevenSeg(HERA_STEER_FAIL);
-        printf("UpdateHeraSteer Failed\r\n");
+        //printf("UpdateHeraSteer Failed\r\n");
```
Comment (nicholassheffler) [inline olympus/main.c]: Are these print statements for debugging?. Do we still need them?

File: olympus/drivers/comms.c [hunk 2]
```diff
@@ -61,7 +61,7 @@ void toggleLED3(Packet_t* packet)
 /** XXX - Remove these when they are reimplemented **/
 static uint8_t status[5] = {0};
 
-static hephaestusData_t hephaestusData;
+//static hephaestusData_t hephaestusData;
```
Comment (nicholassheffler) [inline olympus/drivers/comms.c]: Is this commented out for a reason?

Comment (Kwuerch) [reply olympus/drivers/comms.c]: Good catch, should be removed.

File: olympus/main.c (hunk 1 above)
Comment (Kwuerch) [reply olympus/main.c]: This shouldn't be commented out, will fix.

Comment (nicholassheffler) [reply olympus/main.c]: There's another one right below it as well.

**Pull request merged**